src/redux/sagas/democracy.js

```diff
@@ -23,7 +23,7 @@ import {
 
 // WORKERS
 
-function getProposers(crossReferencedList, isProposer = false) {
+function getProposers(crossReferencedList = [], isProposer = false) {
```

Comment:
I believe this is leftover from your local testing

Reply:
Done!